vivisect/analysis/i386/calling.py

```diff
@@ -113,13 +113,12 @@ def analyzeFunction(vw, fva):
     cc = emu.getCallingConvention(callconv)
     stcount = cc.getNumStackArgs(emu, argc)
     stackidx = argc - stcount
+    baseoff = cc.getStackArgOffset(emu, argc)
```

Comment:
could.  i was just trying to make use of the value that calling conventions already provide with CC_STACK_INF, although it is less efficient.  i'll make the change and let you know.